dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -306,6 +306,28 @@ public abstract class Entry<S extends Entry, T extends Version> implements Compa
     @Transient
     private Set<OrcidAuthor> orcidAuthors = new HashSet<>();
 
+    @JsonIgnore
+    @Column(nullable = true, columnDefinition = "varchar(32)")
+    @Schema(description = "The visibility of the Git repo")
```

Comment:
If it's not intended to be returned, does it need to be described in the openapi?